### ISSUE TITLE
clearpath_diagnostics not available in binaries in Humble

### ISSUE THREAD

**Issue by SteveMacenski** [clearpathrobotics/clearpath_robot#58]
**Please provide the following information:**
 - OS: 22.04 
 - ROS 2 Distro: Humble
 - Built from source or installed: ROS apt repo installed
 - Package version: NA
 - Real hardware or simulation: NA

 **Expected behaviour**

```
sudo apt install ros-humble-clearpath-diagnostics
```

It is expected that I can install all clearpath software without source builds

 **Actual behaviour**

It is not released in binary format and without builds in the ROS build farm at build.ros2.org.

**Comment by SteveMacenski** [comment]
It also appears that there is a dependency on the `micro_ros_agent` that is similarly not available in binary form

**Comment by tonybaltovski** [comment]
The robot packages are the only ones not on the OSRF buildfarm since they depend on things that are not released or closed source packages.  Would adding the Clearpath package server work for getting around this? https://wiki.ros.org/ClearpathRobotics/Packages

**Comment by SteveMacenski** [comment]
I appear (and to be perfectly frank, I'm still at the phase of restructuring the packages and making sure things seem to generally work at an Alpha level with limited success, so very well could hit that wall) to be able to build the diagnostic package here without any proprietary software and I see the battery control & estimator nodes run, so that seems to work fine? 

I did not try to compile the other packages here though, just the clearpath_diagnostics. If that one is OK, you could release that to the public build farm alone.

**Comment by tonybaltovski** [comment]
The idea was to keep `clearpath_diagnostics` in `clearpath_robot` since it only needs to ran on the actual robot.  The battery control & estimator nodes will run without any packages from the Clearpath Package server.  Is it that you do not want to add the Clearpath package server to run the platform or would you want the diagnostics for simulations are well?

**Comment by SteveMacenski** [comment]
> Is it that you do not want to add the Clearpath package server to run the platform or would you want the diagnostics for simulations are well?

Its more that other `clearpath-*` are available and this seems like one that can be released so that it all works. So far, that's the only Clearpath package I've run into that I had to build from source in my workspace to run on my developer computer for simulation and testing. Since this doesn't tie into the proprietary systems, I think this could be released in the public build farm. 

The Clearpath package server is fine and I have no problem with that. I think this was also filed before I actually knew that package server existed (the provided documentation in the box didn't mention this and I guess I didn't run into it in the tutorials/documentation pages I reviewed).
